pandas/core/series.py

```diff
@@ -1658,7 +1658,7 @@ def unique(self):
             * Datetime with Timezone
             * Interval
             * Sparse
-            * IntegerNA
+            * IntegerNA.
```

Comment:
I suppose this "works" but seems hacky and rather long for a return. Can you move some of this into a `Notes` section instead?

Reply:
agreed, I put the basic return documentation in the Return section. Everything else moved to a new Notes section

Reply:
Looks good but can remove trailing period now